scripts/owl_reader.py

```diff
@@ -0,0 +1,104 @@
+import owlready2 as owl
+import time
+import sys
+from collections import OrderedDict
+import re
+
+class OwlReader:
```

Comment:
could you add a small comment to this file about what the purpose of the script is?

Reply:
Sure